wds-required-plugins.php

```diff
@@ -112,11 +112,6 @@ public function required_text_markup() {
 	 * @return array
 	 */
 	public function filter_plugin_links( $actions = array(), $plugin ) {
-		// Remove edit link for all plugins
-		if ( array_key_exists( 'edit', $actions ) ) {
-			unset( $actions['edit'] );
-		}
-
```

Comment:
We could put it back, but is removing the edit link from all plugins actually relevant to this specific "required plugins" plugin? Maybe we only apply it to the required plugins instead?